### DISCUSSION TITLE
Game Settings, add an option to adjust the growth of level 1 creatures in relation to the base growth of the creatures

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#3603] [Ideas]
Option: "Castle: Adjust the number of creatures generated by the Peasants' and Goblins' growth building in relation to the base growth".

I suggest that in the Knight's and Barbarian's castles, the building that adds growth to the first-level creatures add a number of creatures corresponding to the base growth of the creatures.

Thus, the "Farm" should increase the growth by 12 Peasants, and the "Garbage Heap" should increase the growth by 10 Goblins.

In fheroes2:

![Level 1 creature growth building fh2 01](https://user-images.githubusercontent.com/43583994/97115484-48db5900-16f7-11eb-99bc-29c4bfb32a5a.png)

![Level 1 creature growth building fh2 02](https://user-images.githubusercontent.com/43583994/97115487-4aa51c80-16f7-11eb-9687-6c3c8d2136d1.png)

**Comment by Branikolog** [comment]
@LeHerosInconnu 
So do you suggest to increase the growth of this two creatures? Isn't it a changing of an original mechanics of the game?

**Comment by LeHerosInconnu** [comment]
@Branikolog.

> @LeHerosInconnu
> So do you suggest to increase the growth of this two creatures? Isn't it a changing of an original mechanics of the game?

Yes, that's why it's optional. :)

All other level 1 creatures have a base growth of 8, except these two.

**Comment by Branikolog** [comment]
@LeHerosInconnu 
:)
For an original gameplay we shouldn't add growth. And for our future addon we would balance 1st level creatures in more special way. )

**Comment by LeHerosInconnu** [comment]
@Branikolog.

> @LeHerosInconnu
> :)
> For an original gameplay we shouldn't add growth. And for our future addon we would balance 1st level creatures in more special way. )

Is this more special way to balance 1st level creatures of the future addon highly secret? :)

**Comment by Branikolog** [comment]
We haven't decided yet. But even in addon we won't change the existing mechanics, especially growth of creatures.

**Comment by ihhub** [comment]
Moving this issue into Discussion area under Ideas section.

**Comment by LeHerosInconnu** [comment]
This is planned in my mod: https://github.com/ihhub/fheroes2/discussions/10091.
I close the discussion.
